tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AdoNet/Oracle/OracleDefinitions.cs

```diff
@@ -11,30 +11,13 @@
 [assembly: AdoNetClientInstrumentMethods(
     AssemblyName = "Oracle.ManagedDataAccess",
     TypeName = "Oracle.ManagedDataAccess.Client.OracleCommand",
-    MinimumVersion = "4.122.0",
-    MaximumVersion = "4.122.*",
-    IntegrationName = nameof(IntegrationId.Oracle),
-    DataReaderType = "Oracle.ManagedDataAccess.Client.OracleDataReader",
-    DataReaderTaskType = "System.Threading.Tasks.Task`1[Oracle.ManagedDataAccess.Client.OracleDataReader]",
-    TargetMethodAttributes = new[]
-    {
-        // int Oracle.ManagedDataAccess.Client.OracleCommand.ExecuteNonQuery()
-        typeof(CommandExecuteNonQueryAttribute),
-        // OracleDataReader Oracle.ManagedDataAccess.Client.OracleCommand.ExecuteReader()
-        typeof(CommandExecuteReaderAttribute),
-        // OracleDataReader Oracle.ManagedDataAccess.Client.OracleCommand.ExecuteReader(CommandBehavior)
-        typeof(CommandExecuteReaderWithBehaviorAttribute),
-        // DbDataReader Oracle.ManagedDataAccess.Client.OracleCommand.ExecuteDbDataReader(CommandBehavior)
-        typeof(CommandExecuteDbDataReaderWithBehaviorAttribute),
-        // object Oracle.ManagedDataAccess.Client.OracleCommand.ExecuteScalar()
-        typeof(CommandExecuteScalarAttribute),
-    })]
-
-[assembly: AdoNetClientInstrumentMethods(
-    AssemblyName = "Oracle.ManagedDataAccess",
-    TypeName = "Oracle.ManagedDataAccess.Client.OracleCommand",
+    // The netframework nuget depends on the v4.122.* of the dll.
+    // It seems that the pattern is 4.122.<major version of the nuget>.
+    // For netcore, the major version of the dll matches the major version of the nuget,
+    // that was v2 and v3, but they have recently bumped it from 3 to 23
+    // (to have matching version numbers between netcore and netframework I suppose).
     MinimumVersion = "2.0.0",
-    MaximumVersion = "2.*.*",
+    MaximumVersion = "23.*.*",
```

Comment:
given the current pattern:
v2, v3, v23 = netcore
v4.122 = netframework
Do you think this min/max is ok, or should we split it in 2 to exclude the range [5-22] (we don't know what they're going to do if they want to bump the version from 4.122) ?

Reply:
I'm torn honestly 😅 I would be inclined to play it safe and split, but that may well be unnecessary.

Reply:
ok, I can split, tbh it's not adding that much complexity